deep/settings.py

```diff
@@ -198,6 +198,9 @@
         'NAME': 'django.contrib.auth.password_validation.'
         'NumericPasswordValidator',
     },
+    {
+        'NAME': 'user.validators.CustomMaximumLengthValidator',
```

Comment:
Could you add a note that we are using the Django admin panel for password reset/change here?